src/text.rs

```diff
@@ -362,6 +362,30 @@ pub fn draw_text_ex(text: &str, x: f32, y: f32, params: TextParams) {
     }
 }
 
+/// Draw multiline text with the given font_size and color.
+pub fn draw_multiline_text(text: &str, x: f32, y: f32, font_size: f32, color: Color) {
+    draw_multiline_text_ex(
+        text,
+        x,
+        y,
+        TextParams {
+            font_size: font_size as u16,
+            font_scale: 1.0,
+            color,
+            ..Default::default()
+        },
+    )
+}
+
+/// Draw multiline text with custom params such as font, font size and font scale.
+pub fn draw_multiline_text_ex(text: &str, x: f32, mut y: f32, params: TextParams) {
+    for line in text.lines() {
+        let dimensions = measure_text(line, params.font, params.font_size, params.font_scale);
+        draw_text_ex(line, x, y, params.clone());
+        y += dimensions.height + dimensions.offset_y;
```

Comment:
should `offset_y` be accumulated for each line? I did not tested it, but I thought `offset_y` is more about where the text is rendered and doesn't affect total line height?

Reply:
Here are some different approaches I tried now instead of adding the offset:
![image](https://github.com/not-fl3/macroquad/assets/85427285/46679d9c-0d02-485a-b13a-061615e7acf5)

* White is the current version.
* Red is the current version, but without adding `offset_y`.
* Green is this function:
 ```rust
pub fn draw_multiline_text_ex3(text: &str, x: f32, mut y: f32, params: TextParams) {
    let max_height = text
        .lines()
        .map(|line| measure_text(line, params.font, params.font_size, params.font_scale).height)
        .fold(0.0, f32::max);

    for line in text.lines() {
        draw_text_ex(line, x, y, params.clone());
        y += max_height;
    }
}
```
It calculates the biggest height first so that the distance between two adjacent lines is always the same.

* Yellow is the same as Green, but the function takes an extra argument `line_distance: f32` that is added to `y` in each iteration: `y += max_height + line_distance;`.

I think Green or Yellow are preferable over White or Red.